### ISSUE TITLE
Autosubmit error messages

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#755]
In GitLab by @mcastril on Nov 30, 2021, 15:51

When I try to monitor an experiment that does not belong to me, I see:

```
autosubmit monitor -txt a44a
[ERROR] Trace: 
 [CRITICAL] Unhandled error: If you see this message, please report it in Autosubmit's GitLab project
```

@dbeltrankyl do you think that you could improve the error message?

If I try to create an experiment that is not mine, I see:

```
mcastril@ bscearth396:/esarchive/autosubmit> autosubmit create a44a
[ERROR] Trace: You don't own the experiment a44a.
 [CRITICAL] User or owner does not exists [eCode=7012]
```

The `ERROR` is correct, but the `CRITICAL` is misleading. Do you think that it could be more `general`?

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Dec 2, 2021, 14:30

I fixed these messages and many others ( like per example, issues when wrappers jobs are not in the current jobs conf)

**Comment by kinow** [comment]
In GitLab by @mcastril on Dec 3, 2021, 17:36

Thanks @dbeltrankyl !

**Comment by kinow** [comment]
In GitLab by @mcastril on Dec 16, 2021, 14:54

@ rambika had another unspecific error that we should fix. Apparently, the error was more specific in 3.12.0. Can you add it here?

**Comment by kinow** [comment]
In GitLab by @ rambika on Dec 16, 2021, 17:54

Thanks @mcastril, As I mentioned in the meeting, I was trying to run an auto-caliope experiment with autosubmit 3.13, I have received a critical error, however while running with 3.12 the error showed more detailed information. 

Error with Auto-submit 3.13: 

> "Checking configuration files...
expdef_a46v.conf OK
platforms_a46v.conf OK
jobs_a46v.conf OK
wrappers OK
autosubmit_a46v.conf OK
Configuration files OK
Using project folder: /esarchive/autosubmit/a46v/proj
[ERROR] Trace:
 [CRITICAL] Unhandled error: If you see this message, please report it in Autosubmit's GitLab project"

Error with Auto-submit 3.12: 

> [CRITICAL] Unhandled exception on Autosubmit: [Errno 2] No such file or directory: '/esarchive/autosubmit/a46v/proj/https://earth.bsc.es/gitlab/es/auto-caliope.git/templates/proj_caliope.conf'
Traceback (most recent call last):
  File "/shared/earth/software/autosubmit/3.12.0-fixed-foss-2015a-Python-2.7.9/lib/python2.7/site-packages/autosubmit-3.12.0-py2.7.egg/autosubmit/autosubmit.py", line 445, in parse_args
    args.expand_status, args.notransitive,args.check_wrapper)
  File "/shared/earth/software/autosubmit/3.12.0-fixed-foss-2015a-Python-2.7.9/lib/python2.7/site-packages/autosubmit-3.12.0-py2.7.egg/autosubmit/autosubmit.py", line 2431, in create
    Autosubmit._create_project_associated_conf(as_conf, False, update_job)
  File "/shared/earth/software/autosubmit/3.12.0-fixed-foss-2015a-Python-2.7.9/lib/python2.7/site-packages/autosubmit-3.12.0-py2.7.egg/autosubmit/autosubmit.py", line 2371, in _create_project_associated_conf
    project_destiny)
  File "/shared/earth/software/Python/2.7.9-foss-2015a/lib/python2.7/shutil.py", line 82, in copyfile
    with open(src, 'rb') as fsrc:
IOError: [Errno 2] No such file or directory: '/esarchive/autosubmit/a46v/proj/https://earth.bsc.es/gitlab/es/auto-caliope.git/templates/proj_caliope.conf'


The error was a small mistake in the path. It would be good if new version given error like the previous one.
